flink/src/test/java/org/apache/sedona/flink/FunctionTest.java

```diff
@@ -73,10 +73,23 @@ public void testPointOnSurface() {
         assertEquals("POINT (32 -118)", result.toString());
     }
 
+    @Test
     public void testReverse() {
         Table polygonTable = createPolygonTable(1);
         Table ReversedTable = polygonTable.select(call(Functions.ST_Reverse.class.getSimpleName(), $(polygonColNames[0])));
         Geometry result = (Geometry) first(ReversedTable).getField(0);
         assertEquals("POLYGON ((-0.5 -0.5, 0.5 -0.5, 0.5 0.5, -0.5 0.5, -0.5 -0.5))", result.toString());
     }
+
+    /* ST_SetSrid Has not been implemented in flink yet
+    @Test
+    public void testAsEWKT() {
```

Comment:
Please enable the test case. Why do you call it ReverseTable?

Reply:
Updated. Couldn't find a way to set SRID using flink methods. This test case tests scenario when SRID isn't present. Is that Okay ?